### PR TITLE
feature/rsk-address-rpc

### PR DESCRIPTION
**Pull request by mauricioirace**
Changes:
- Replace SendMessageToTopic calls by SendMessageToRskAddress

### CODE REVIEW

File: proto/api.proto [hunk 1]
```diff
@@ -80,6 +83,12 @@ message PublishPayload {
 }
 
 
+message RskAddressPublish {
+    string address = 1;
```
Comment (santicomp2014) [inline proto/api.proto]: This should use the same message as `RskAddress`
`RskAddress address = 1`

**Review by santicomp2014 — CHANGES_REQUESTED** [review]
Minor change to the proto, everything else is ok.
Thanks

**Review by santicomp2014 — APPROVED** [review]
LGTM thanks for the change

**Review by marcosmartinez7 — APPROVED** [review]

**Review by mortelli — APPROVED** [review]
thanks @mauricioirace for this :+1: 

looks good from my end.

i might have some changes to suggest based on whether or not i get to try out these scripts (and if so, based also on whether i have any changes to suggest).

for now we're good to go

**Pull request merged**